Vault-Deployment/loki-alerting-rules.tf

```diff
@@ -0,0 +1,11 @@
+# Loki alerting and recording rules
+resource "kubernetes_manifest" "loki-alerting-rules" {
+  manifest = yamldecode(file("./manifests/loki-alerting-rules/loki-rules.yaml"))
+  depends_on = [helm_release.loki]
+}
+
+# Prometheus alerting and recording rules
+resource "kubernetes_manifest" "prometheus-alerting-rules" {
+  manifest = yamldecode(file("./manifests/loki-alerting-rules/prometheus-rules.yaml"))
+  depends_on = [helm_release.prometheus-grafana]
```

Comment:
because the promql alerting rules for Grafana are mounted to the Grafana pods, we need to wait for this namespace to be created (not the Loki namespace, as above).